### DISCUSSION TITLE
Offline Mode and Session Token or "How can i access my DLC's in offline mode" | Hunter COTW

### DISCUSSION THREAD

**Discussion by krrrrrrr-sudo** [legendary-gl/legendary#643] [Q&A] [answered]
Hi, thanks for this great launcher. It work's and i get rid of the EGL. One Problem which i also have with EpicGamesLauncher. 
The game "hunter call of the wild" seem to launch properly when i either go offline mode via legendary or EGL. 
But the game can not access the DLC's i bought. It somehow has only some online verification. Is it possible to have, for example, checked the purchased items online and then save a token which ages but not so fast, let's say, 7 days. After that i have to go online and generate a new "inventory" for the game.
I ask, because i don't want to be online all the time and also, the base game is able to launch offline but only with base content.
Are you able to fetch or extract this authentication process and "save" or store it into a file or so ?

**Comment by arielj** [comment]
That's not how DRM works, if the developers made the game expecting you to be online to be able to use the DLCs there's no token or anything locally. that would beat the purpose of the DRM if the check is done only locally without checking the actual servers.

I doubt there's anything you can do if the game doesn't support that use case.

**Reply by krrrrrrr-sudo** [reply]
Thanks and yes, i know that, but that doesn't answer my initial question fully. In other words, is it possible to see the authentification or verification process in the legendary launcher or somehow in the ecom API so that i could have a clue and maybe the possibility to work on this myself ? I know this might be on some point a grayzone but i don't want to crack something or so. I bought the items on the store legally and only want to play offline. If legendary is able to to that but has not implemented it, that's fine.
Also, just to mention: xbox users should be able to play the game offline because there are some discussion boards, which mention the same problem and the solution is there. And i have a ps4 and the base game installed but no dlc available. Maybe i buy one to test offline gaming. So, there is a difference on what platform i play regarding this game. This is not really cool.
UPDATE: so i bought 2 dlc's and their available on ps4 when going offline. I know this is off topic but just wanted to let you know.

**Comment by krrrrrrr-sudo** [comment, marked as answer]
close it.
